### PR TITLE
Use react-scripts with configured webpack\lint

### PR DESCRIPTION
**Pull request by GreyYar**
+divide app into components  
+add start logic  
+add quiz logic (WIP)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  };
+
+  incrementQuestion() {
+    this.setState({ currentQuestion: ++this.state.currentQuestion })
+  }
+
+  generateQuestionsQueue() {
+    this.setState({ visibilityStart: false });
+    let arr = [];
+    if (questions_amount > charBase.length)
```
Comment (artem-malozemov) [inline src/App.js:37]: state.base instead of charBase?

File: src/components/Label.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class Label extends React.Component {
```
Comment (artem-malozemov) [inline src/components/Label.js]: Could you, please, replace it with stateless component?

**Review by artem-malozemov — APPROVED** [review]
Approved with suggestions.
Also - it will be greate, if you will attach gif-animation with work of app :)

File: src/App.js (hunk 1 above)
Comment (GreyYar) [reply src/App.js:37]: Fixed, thanks

File: src/components/Label.js (hunk 2 above)
Comment (GreyYar) [reply src/components/Label.js]: Done

**Pull request merged**